public/js/script.js

```diff
@@ -257,18 +274,169 @@ const streamUnstream = () => {
 const iconStream = (isStream) => {
     if (!isStream) {
         const html = `
-        <i class="fas fa-video-slash active"></i>
+        <i class="fas fa-lg fa-video-slash active"></i>
         <span class="main__control__button__text active">Stream</span>
         `
 
         document.querySelector('#main__control__video').innerHTML = html
     } else {
         const html = `
-        <i class="fas fa-video"></i>
+        <i class="fas fa-lg fa-video"></i>
         <span class="main__control__button__text">Unstream</span>
         `
 
         document.querySelector('#main__control__video').innerHTML = html
     }
 }
 
+
+const myScreen = document.createElement('video')
+myScreen.className = 'screen-video'
+let screenPermission = true
+
+const shareScreen = () => {
+    navigator.mediaDevices.getDisplayMedia({
+        video: {
+            cursor: "always"
+        },
+        audio: {
+            echoCancellation: true,
+            noiseSuppression: true
+        }
+    }).then(stream => {
+        screenStream = stream
+        let screenTrack = stream.getVideoTracks()[0]
+        screenTrack.onended = () => {
+            shareUnshare()
+        }
+
+        iconShareScreen(true) // set icon
+
+        // TODO: Search All Available Peer => 
+        for (let i = 0; i < currentPeer.length; i++) {
+
+            let sender = currentPeer[i].getSenders().find((sender) => {
+                return sender.track.kind == screenTrack.kind
+            })
+
+            sender.replaceTrack(screenTrack)
+        }
+
```

Comment:
This code will only replace the user's media stream from getUserMedia to getDisplayMedia, not open a new peer connection to share screen and share the video at the same time. I call this ugly way but for now, it works like a charm.